client/my-sites/domains/domain-management/dns/dns-menu-options-button.tsx

```diff
@@ -303,12 +303,13 @@ function DnsMenuOptionsButton( {
 			<RestoreDefaultARecordsDialog
 				visible={ isRestoreARecordsDialogVisible }
 				onClose={ closeRestoreARecordsDialog }
-				defaultRecords={ null }
```

Comment:
This was the only instance of RestoreDefaultARecordsDialog I could find, and it's always passing in Null, so I removed it as a prop from the component

Reply:
Good catch!